frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -453,10 +453,12 @@ placeable-parser-xmlTag =
 ## Resource menu
 ## Used in the resource menu in the main navigation bar.
 resource-ResourceMenu--no-results = No results
-resource-ResourceMenu--resource = Resource
-resource-ResourceMenu--progress = Progress
+resource-ResourceMenu-resource = Resource
+resource-ResourceMenu-progress = Progress
```

Comment:
We actually need double dashes here (and in the JS file). It's our convention to split the component name and the ID with a double dash.

Reply:
Oh! I thought it was only when it contains two/more words for the id. I've put it back the double dashes.